doc/Doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = "@dlite_VERSION@"
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = "Lightweight library for working with scientific data"
+PROJECT_BRIEF          = "Lightweight data-centric interoperability framework for working with scientific data"
```

Comment:
This should be updated in `python/setup.py` as well for the short description - and anywhere else that's similar.

Reply:
I am not sure DLite is limited to scientific data. How about branding it as:
"DLite - A lightweight data-centric framework for semantic interoperability"?

Reply:
I am not sure DLite is limited to scientific data. How about branding it as:
"DLite - A lightweight data-centric framework for semantic interoperability"?

Reply:
Done